### PR TITLE
chore(sdk): enhance sdk documentation

### PR DESCRIPTION
**Pull request by cpaulve-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

**Review by kpanot — APPROVED** [review]

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md [hunk 1]
```diff
@@ -127,8 +127,14 @@ You will need to compute this information with the two timezones -- the airport'
 The Otter framework has introduced the `utils.Date` and `utils.DateTime` objects to replace the `Date` implementation and convert the date returned by the API as if it were in the
 timezone of the user.
 
-Dates can be generated as `utils.Date` or `string` depending on the value of the `stringifyDate` option. This ensures that the timezone will not impact the date. 
-In the case of `date-time` objects, the default type used is the native `string` and `Date` type depending on the `stringifyDate` option value.
+#### Date format
+Dates can be generated as utils.Date or string depending on the value of the stringifyDate option.
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md:131]:
```suggestion
Dates can be generated as `utils.Date` or `string` depending on the value of the `stringifyDate` option.
```

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md [hunk 2]
```diff
@@ -127,8 +127,14 @@ You will need to compute this information with the two timezones -- the airport'
...
+This ensures that the timezone will not impact the date as the native Date object does not handle the 'dd/mm' format well.
+
+#### Date-time format
+In the case of `date-time` objects, the default type used is the native string or the native `Date` type depending on the `stringifyDate` option value.
+If you want to generate a `date-time` using `utils.DateTime` presented in the [previous section](#the-timezone-issue),
+you can do it at property level thanks to the x-local-timezone vendor.
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md:137]:
```suggestion
you can do it at property level thanks to the `x-local-timezone` vendor.
```

Comment (sdo-1A) [reply packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md:137]: isn't this a repetition of the next line ? (line 139)

File: packages/@ama-sdk/schematics/README.md [hunk 3]
```diff
@@ -132,6 +132,9 @@ If your specification file includes dates, there are multiple options for the ge
   For more information related to these types, check out this [documentation](https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates).
   This can be done by adding `--global-property stringifyDate=false` to the generator command or by adding the global property
 to the `openapitools.json`.
+- Note that an extra type to manage special timezone uses case can be used at property level thanks to the
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/README.md:135]:
```suggestion
- Note that an extra type to manage special timezone use cases can be used at property level thanks to the 
```

File: packages/@ama-sdk/schematics/README.md [hunk 4]
```diff
@@ -132,6 +132,9 @@ If your specification file includes dates, there are multiple options for the ge
   For more information related to these types, check out this [documentation](https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates).
   This can be done by adding `--global-property stringifyDate=false` to the generator command or by adding the global property
 to the `openapitools.json`.
+- Note that an extra type to manage special timezone uses case can be used at property level thanks to the 
+`x-local-timezone` vendor.
+Please check out the [date documentation](https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates)
```
Comment (sdo-1A) [inline packages/@ama-sdk/schematics/README.md:137]:
```suggestion
Please check out the [date documentation](https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates).
```

Comment (matthieu-crouzet) [inline packages/@ama-sdk/schematics/README.md:137]:
```suggestion
> [!NOTE]
> An extra type to manage special timezone use cases can be used at property level thanks to the 
> `x-local-timezone` vendor.
> Please check out the [date documentation](https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates)
```

**Review by kpanot — APPROVED** [review]

**Pull request merged**